frame/proxy/src/lib.rs

```diff
@@ -165,9 +220,142 @@ decl_module! {
 		/// The maximum amount of proxies allowed for a single account.
 		const MaxProxies: u16 = T::MaxProxies::get();
 
+		/// `MaxPending` metadata shadow.
+		const MaxPending: u32 = T::MaxPending::get();
+
+		/// `AnnouncementDepositBase` metadata shadow.
+		const AnnouncementDepositBase: BalanceOf<T> = T::AnnouncementDepositBase::get();
+
+		/// `AnnouncementDepositFactor` metadata shadow.
+		const AnnouncementDepositFactor: BalanceOf<T> = T::AnnouncementDepositFactor::get();
+
+		fn on_runtime_upgrade() -> Weight {
+			Proxies::<T>::translate::<(Vec<(T::AccountId, T::ProxyType)>, BalanceOf<T>), _>(
```

Comment:
Needs a proper review as there isn't a unit test (though we can do proper integration testing on Westend)

Reply:
It looks right afaict.